subprojects/integ-test/src/integTest/groovy/org/gradle/integtests/StaleOutputHistoryLossIntegrationTest.groovy

```diff
@@ -58,7 +58,6 @@ class StaleOutputHistoryLossIntegrationTest extends AbstractIntegrationSpec {
         result = runWithMostRecentFinalRelease(JAR_TASK_NAME)
 
         then:
-        javaProject.assertBuildTasksExecuted(result)
```

Comment:
:o: The changes in this file can't be matched to the original PR https://github.com/gradle/gradle/pull/1952. Could you explain those changes?